app/controllers/jobseekers/job_applications/build_controller.rb

```diff
@@ -17,7 +17,7 @@ def update
       job_application.update(update_params)
       redirect_to jobseekers_job_applications_path, success: t("messages.jobseekers.job_applications.saved")
     elsif form.valid?
-      if referrer_is_finish_wizard_path?
+      if redirect_to_review?
```

Comment:
is it worth  breaking this into a few methods? as soon as i see nested if/else/elseif type things it makes me think breaking it down worth thinking about

Reply:
I agree, but I just hadn't dedicated any brain time to thinking about doing it any differently. Let me have a think

Reply:
What do you think of those changes?

Reply:
yeah i find it little easier to understand the flow of whats going on. potentially easier to unit test but realise its a controller so you wouldnt go there neccessarily